composer.json

```diff
@@ -10,11 +10,12 @@
     ],
     "license": "BSD-3-Clause",
     "require": {
-        "silverstripe/framework": "^4.0",
-        "silverstripe/admin": "^1.0"
+        "php": "^7.4 || ^8.1",
```

Comment:
sorry one more :) you can use cms4 on 8.0 so we should have ^8.0 :)

Reply:
Sweet, I've updated it🌟